server/socket.js

```diff
@@ -0,0 +1,11 @@
+const socket = require('socket.io');
+
+module.exports = function configureSocket(server) {
+  const io = socket(server);
+
+  io.on('connection', function(socket){
+    socket.on('chat message', function(msg){
+      io.emit('chat message', `Echo: ${msg}`);
```

Comment:
Currently server sends received message to all clients, but instead it should reply to the sender only. Steps to reproduce:
1. Start server
2. Open 2 browser windows
3. Send message from one of the windows
4. Observe this message in another browser window

Reply:
everithing is ok